plugin/audit_log/audit_log.c

```diff
@@ -264,6 +286,22 @@ char *escape_string(const char *in, size_t inlen,
     if (endptr)
       *endptr= out + 1;
   }
+  if (full_outlen)
+  {
+    char tmp[20];
+    in= in + inlen;
+    *full_outlen= *full_outlen + outlen;
+    inlen_orig= inlen_orig - inlen;
+    while (inlen_orig > 0)
+    {
+      size_t tmp_size= sizeof(tmp);
+      inlen= inlen_orig;
+      format_escape_func[audit_log_format](in, &inlen, tmp, &tmp_size);
+      in= in + inlen;
+      inlen_orig= inlen_orig - inlen;
+      *full_outlen= *full_outlen + outlen;
```

Comment:
Good catch! Actually adding outlen could even make harm in case of short strings!